CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 5.2.0 (unreleased)
+Extends support of the SDK to OneView REST API version 1600 (OneView v5.20).
```

Comment:
Add the list of resources supported in this version

Reply:
done